packages/data-explorer/src/components/SearchComponent.vue

```diff
@@ -50,12 +50,13 @@ export default {
   methods: {
     handleSearchAction () {
       this.$emit('input', this.searchText)
-      createBookmark(this.$router)
     }
   },
   watch: {
     value: {
       handler: function (val, oldVal) {
+        this.searchText = val
+
         if (val === '') {
           this.searchText = ''
         }
```

Comment:
this.searchText is set to '' when val === '' , no need to do it again on line 61

Reply:
Reverted changes